components/scream/src/physics/rrtmgp/simple_netcdf.hpp

```diff
@@ -0,0 +1,291 @@
+#ifndef SIMPLE_NETCDF_HPP
```

Comment:
Not essential for this PR to go in, but should we include a comment here that lets any new developer know that this code is in place for needs particular to RRTMGP, and that any standard IO should use the scorpio interface?

I forsee a distant future where someone greps for netCDF and stumbles upon this code, and as a result bends over backwards to get their IO to work with this interface instead of scorpio.

Reply:
Sure, but I'd think that the fact that it exists in the `rrtmgp` subdirectory makes it pretty clear that it exists for RRTMGP and not necessarily for the rest of the model. Alternatively, we could probably rename this to something like rrtmgp_netcdf or something to make it clear that it exists for RRTMGP if you think it's existence in the `rrtmgp` subdirectory is not enough to clearly distinguish it from general SCREAM codes.

Reply:
Added comment anyways...